.github/workflows/test.yml

```
@@ -0,0 +1,63 @@
+        run: flutter pub get
+
+      - name: Analyze code
+        run: flutter analyze
+
+      # 生成测试覆盖率报告
+      - name: Run tests with coverage
+        run: flutter test --coverage
+
+      # 上传覆盖率报告到 Codecov
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)